packages/api-client/.eslintrc.json

```diff
@@ -4,6 +4,7 @@
         "@typescript-eslint/no-explicit-any": "off",
         "@typescript-eslint/no-unused-vars": "off",
         "@typescript-eslint/explicit-module-boundary-types": "off",
-        "import/extensions": "off"
+        "import/extensions": "off",
+        "import/prefer-default-export": "off"
```

Comment:
Else I get the linting error: `Prefer default export  import/prefer-default-export` in the `ApiClientFactory.ts` file